src/main/java/org/apache/maven/plugins/enforcer/EnforceBytecodeVersion.java

```diff
@@ -308,7 +308,7 @@ private String isBadArtifact( Artifact a )
         }
         catch ( IOException e )
         {
-            throw new EnforcerRuleException( "IOException while reading " + jarFile.getName(), e );
+            throw new EnforcerRuleException( "IOException while reading " + f.getAbsolutePath(), e );

```

Comment:
BTW just

```java
throw new EnforcerRuleException( "IOException while reading " + f, e );
```

suffices since `File.toString` returns the path.

(And if anyone is actually constructing `File`s using path names relative to the process working directory, they deserve everything they get. I do not think Maven would do that.)